app/views/shared/_api_selector.html.slim

```diff
@@ -1,6 +1,9 @@
 = javascript_pack_tag 'api_selector'
 
+- settings_link = can?(:manage, current_account) ? provider_admin_account_path : edit_provider_admin_user_personal_details_path
```

Comment:
Is there any reason why `settings_link` is not defined in [menu_helper.rb](https://github.com/3scale/porta/blob/master/app/helpers/menu_helper.rb), as `current_api`, for example?

Reply:
Ah, you're right it should be there